scripts/get_untriaged.py

```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8 -*-
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import argparse
+import json
+import sys
+from datetime import date
+from datetime import timedelta
+
+# Inject project path
+sys.path.append('../')
+
+
+def get_untriaged(args):
+    from bugbug import bugzilla
+
+    today = date.today()
+    three_months_ago = today - timedelta(days=args.days_back)
+
+    # Set bugzilla token and download bugs
+    bugzilla.set_token(args.token)
+    bugzilla.download_bugs_between(three_months_ago, today)
+
+    # Get untriaged bugs
+    bugs = bugzilla.get_bugs()
+    untriaged_bugs = []
+    for bug in bugs:
+        for history in bug['history']:
+            for change in history['changes']:
+                if change['field_name'] == 'component' and change['added'] == 'Untriaged':
+                    untriaged_bugs.append(bug)
+
+    return json.dumps(untriaged_bugs)
+
+
+if __name__ == '__main__':
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--token',
+                        help='Bugzilla API token')
+    parser.add_argument('--days-back',
+                        type=int,
+                        help='Days since to fetch bugs')
+
+    args = parser.parse_args()
+    print(get_untriaged(args))
```

Comment:
It'd be nice, after getting the untriaged bugs, to apply the models we have for triage and calculate the metrics for them.